spec/spec_helper_local.rb

```diff
@@ -60,117 +60,226 @@
 shared_context 'Fedora 17' do
```

Comment:
I'd consider dropping this and opened https://github.com/puppetlabs/puppetlabs-apache/pull/2238. You may want to merge that first and rebase on it.